libraries/AP_RangeFinder/AP_RangeFinder_Benewake_TFMiniPlus.cpp

```diff
@@ -102,6 +104,8 @@ bool AP_RangeFinder_Benewake_TFMiniPlus::init()
     if (val[5] * 10000 + val[4] * 100 + val[3] < 20003) {
         hal.console->printf(DRIVER ": minimum required FW version 2.0.3, but version %u.%u.%u found\n",
                             val[5], val[4], val[3]);
+        gcs().send_text(MAV_SEVERITY_INFO, ": minimum required FW version 2.0.3, but version %u.%u.%u found\n",
```

Comment:
We probably want to step up the severity a little.  You can find the list in `common.xml` if you search for `MAV_SEVERITY_INFO`.

Reply:
yes, let's make it MAV_SEVERITY_CRITICAL.

Reply:
@peterbarker could you please review the changes